Nodejs/Product/Nodejs/ProjectTemplates/TypeScriptAzureExpressApp/TypeScriptExpressApp.njsproj

```diff
@@ -31,10 +28,10 @@
     <DebugSymbols>true</DebugSymbols>
   </PropertyGroup>
   <ItemGroup>
-    <TypeScriptCompile Include="app.ts" />
-    <TypeScriptCompile Include="routes\index.ts" />
-    <TypeScriptCompile Include="routes\user.ts" />
-    <Content Include="tsconfig.json" />
+    <None Include="app.ts" />
+    <None Include="routes\index.ts" />
+    <None Include="routes\user.ts" />
+    <None Include="tsconfig.json" />
     <Content Include="package.json" />
```

Comment:
>Content [](start = 5, length = 7)

`None`?

Reply:
I'm inclined to say it's better to use `None` for `package.son` and `readme.md`. I'll track this on issue #2128